src/Annotations/InjectedFromContext.php

```diff
@@ -0,0 +1,17 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\RequestHandlers\Annotations;
+
+/**
+ * This annotation is used on properties to indicate that they should only be injected from
+ * the context of the request and not be processed from the request body.
```

Comment:
Maybe replace 'not processed from the request body' with 'not populated with data from the request body'